.github/workflows/test-integrations.yml

```diff
@@ -305,7 +305,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        envoy-version: ["1.28.3"]
+        envoy-version: ["1.29.5"]
```

Comment:
Should this be `1.28.4`? It [looks like](https://developer.hashicorp.com/consul/docs/connect/proxies/envoy#standard-releases) Consul 1.18 doesn't support Envoy `1.29.5`